docs/stencil-docs/customizing-checkout/checkout-sdk-example.md

```diff
@@ -116,421 +116,20 @@ lineNumbers: true
 extensions: ['.js', '.jsx']
 ```
 
-### Add React Checkout Components to Theme
+### Creating a checkout-loader.js file

```

Comment:
We have got this ticket https://jira.bigcommerce.com/browse/CHECKOUT-5191 to provide the checkout sdk with an checkout loader file. We shouldn't need to explain how to create the checkout loader file, it should come with the code, same way the open source checkout does it today.

Only instructions are:
1. Install dependencies (npm ci)
2. Build checkout (npm build)
3. Upload checkout-loader file
4. Enter checkout-loader URL in Control Panel

Sorry for the confusion, we have been discovering missing bits and pieces along the way, and the Checkout SDK Example is one of them.